### DISCUSSION TITLE
Full documentation where?

### DISCUSSION THREAD

**Discussion by fuxoft** [8VIM/8VIM#540] [Q&A]
Hello, I am unable to find the full docs for 8Vim. I've looked at Github "Usability Guide" and I see no mention of what the "Fn" and "Ctrl" keys do. Also, when I click the emoji symbol, 8VIM switches to another keyboard (and displays no emoji).

**Comment by MaethorNaur** [comment]
I'm building a website with the docs: https://8vim.github.io
If that's more helpful.
For the `emoji symbol` it should switch to the `emoji keyboard` defined in the settings. (it's not in the docs yet :( )

**Reply by MaethorNaur** [reply]
Also, there's an issue for the built-in emoji in the keyboard without switching the keyboard: https://github.com/8VIM/8VIM/issues/435.

Personally, I'm in favour of built-in emoji rather than switching to an external keyboard

**Reply by fuxoft** [reply]
> For the `emoji symbol` it should switch to the `emoji keyboard` defined in the settings. (it's not in the docs yet :( )

On my Pixel 6a, in the settings, I can select "Google Voice Typing" or "Google GBoard" for the "emoji keyboard". Later, when I press the emoji button, 8VIM switches to either Google Voice Typing or Google GBoard, **without displaying any emojis**. It just switches the keyboard and I have to switch back to 8VIM using the built in system keyboard switcher. Is this how it's supposed to work?

**Reply by MaethorNaur** [reply]
Yes, it switches to the keyboard set in the config
